package/benchmarks/wfg/_wfg.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from typing import Any
+
+import optproblems.wfg
+import optuna
+import optunahub
+
+
+class Problem(optunahub.benchmarks.BaseProblem):
+    """Wrapper class for the WFG test suite of optproblems."""
+
+    def __init__(
+        self, function_id: int, num_objectives: int, num_variables: int, k: int, **kwargs: Any
```

Comment:
What about making `k` optional as the original author has his recommendation?
Plus, I think it would be more friendly to use the Optuna conventional variable names such as `n_objectives` and `dimension` rather than `num_objectives` and `num_variables`.

Reply:
> What about making k optional as the original author has his recommendation

Thank you for your feedback. I introduced the default.

> Plus, I think it would be more friendly to use the Optuna conventional variable names such as n_objectives and dimension rather than num_objectives and num_variables.

This wrapper provides `__getattr__` to allow users access to the optproblems' properties. So, I want to keep the original property names to avoid confusion.
```python
    def __getattr__(self, name: str) -> Any:
        return getattr(self._problem, name)
```

```python
wfg = optunahub.load_module("benchmarks/wfg")
wfg4 = wfg.Problem(function_id=4, num_objectives=2, num_variables=3, k=1)

print(wfg4.num_variables)  ## Users can access the optproblems' properties
```

Reply:
Hm, the only problem is that they are not really documented in [the original doc](https://www.simonwessing.de/optproblems/doc/wfg.html) either...

Reply:
Some common properties are defined in `optproblems.base.Problem`.
https://www.simonwessing.de/optproblems/doc/base.html

Reply:
Tbh, the attribute access to the wrapped modules is optional and it is very annoying to change the variable name for each problem registered in OptunaHub, so I highly prefer the same variable names over benchmark problems in OptunaHub.

Example:

```python
import itertools

dimension = 2
problems = [
    optunahub.load_module("benchmarks/bbob").Problem,
    optunahub.load_module("benchmarks/wfg").Problem,
]
for problem_cls in problems:
    problem = problem_cls(
        function_id=function_id,
        dimension=dimension,
        **kwargs,
    )
    # >>> Fail here because of the signature inconsistency.
```

Reply:
OK. Applied the comment.